### PR TITLE
feat: CircleCI continuous deployment to Divshot

### PR DESCRIPTION
**Pull request by laurendorman**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+test:
+  override:
+    - ./node_modules/.bin/grunt ci
+
+deployment:
+  development:
+    branch: master
+    commands:
+      - npm install -g divshot-cli
+      - divshot push development --token $DIVSHOT_TOKEN
```
Comment (danrosenthal) [inline circle.yml]: @laurendorman do we need to fill in the divshot token here?

**Comment by laurendorman** [comment]
@danrosenthal, nope! $DIVSHOT_TOKEN is pulled in from CircleCI's environment variables.

**Comment by danrosenthal** [comment]
@laurendorman cool! Is this safe to merge?

**Comment by danrosenthal** [comment]
screenshot 2015-05-05 09 45 16
Looks like there might be a typo here. That's your failing test.

**Pull request merged**